pkg/reconciler/inmemorychannel/dispatcher/controller.go

```diff
@@ -74,15 +73,26 @@ func NewController(
 	inMemoryDispatcher := inmemorychannel.NewDispatcher(args)
 
 	inmemorychannelInformer := inmemorychannelinformer.Get(ctx)
+	informer := inmemorychannelInformer.Informer()
 
 	r := &Reconciler{
 		Base:                    base,
 		dispatcher:              inMemoryDispatcher,
 		inmemorychannelLister:   inmemorychannelInformer.Lister(),
-		inmemorychannelInformer: inmemorychannelInformer.Informer(),
+		inmemorychannelInformer: informer,
 	}
 	r.impl = controller.NewImpl(r, r.Logger, ReconcilerName)
 
+	// Nothing to filer on, enqueue all imcs if configmap updates.
+	noopFilter := func(interface{}) bool { return true }
+	resync := configmap.TypeFilter(channel.EventDispatcherConfig{})(func(string, interface{}) {
+		r.impl.FilteredGlobalResync(noopFilter, informer)
```

Comment:
maybe use GlobalResync?

Reply:
I renamed to `resyncIMCs` to be more specific.